code/transportation-development-services/tds.js

```diff
@@ -625,6 +634,39 @@ $(document).on('knack-view-render.view_744', function(event, view, record) {
       ${dropdownMenuItem(recordId, "add-tia-communication", "fa-plus-circle", "Communication")}\
     </ul>\
   </div>`).appendTo("#view_744")
+
+  // *** Dropdown menu for mobile views ***
+  $(`<div class="mobile-details-dropdown-menu">\
+    <ul id="tia-mobile-menu-list">\
+      <li class="tia-mobile-dropdown-menu">\
+        <span class="tia-button mobile-dropdown-button"> \
+          <i class="fa fa-angle-down tia-dropdown" /> \
+          Case Management Menu\
+        </span>\
+        <ul class="tia-dropdown-menu-list" style="min-width: 152px; margin: .5em;">\
+          ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details", false, true)}\
```

Comment:
like I said in the description, we can swap the position of the last two variables in the dropMenuItem function, and then it would be 
` ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details", true)}` for mobile and 
` ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details", false, true)}` for desktop newTabs
instead of the way it is now.

Reply:
I agree, lets switch so we dont have to pass the newTab argument if not necessary